source/blog/2016-02-24-devconf-2016.html.md

```diff
@@ -0,0 +1,42 @@
+---
+title: My Devconf.CZ 2016 experience
+author: ykaul
+tags: community, news, blog
+date: 2016-03-07 15:00:00 CET
+---
+
+On the first weekend of February I had the pleasure of attending DevConf.CZ 2016, which took place in the wonderful city of Brno, Czech Republic.
+It's a relaxed, young and vibrant conference and it was fun and rewarding from my perspective.
+Here's a disorganized personal summary:
+
+- Met many community members from all over Europe. Was excited to see so many happy users of the project!
+- Attended [in-depth look of virtual machine migration algorithms - Marcelo Tosatti](https://www.youtube.com/watch?v=XkMIMJKJeTY "in-depth look of virtual machine migration algorithms - Marcelo Tosatti"). I hope to see some of it in coming versions of QEMU, while we work on other improvements to this critical feature.
+- [Growing the ARM server ecosystem - Jim Perrin](https://www.youtube.com/watch?v=q4gU87wdRtI "Growing the ARM server ecosystem - Jim Perrin") was interesting. Seems like a great positive effort, but the road is still long for complete support. Would be cool to support ARM based hosts running virtual machines!
+- Attended [Qemu Disk I/O: Which performs better, Native or Threads? - Pradeep K Surisetty](https://www.youtube.com/watch?v=Jx93riUF5_I "Qemu Disk I/O: Which performs better, Native or Threads? - Pradeep K Surisetty"). The results were not conclusive, though some improvements in native brought it close to or better in some cases to threads. Seems that our default of heuristic for threads in file-based storage and native in block based storage is fine for the time being, but we'll watch closely for developments in this area.
```

Comment:
Edits:

* brought it close to or better in some cases to threads > brough it close to, or in some cases better than threads